asm/scrcmd_pokemon_misc.s

```diff
@@ -2928,11 +2928,11 @@ _02202224:
 	mov r1, #0
 	str r1, [sp]
 	str r1, [sp, #4]
-	mov r1, #0xac
-	mov r2, #4
-	mov r3, #1
+	mov r1, #SPECIES_PICHU
+	mov r2, #4 ; NATURE_NAUGHTY
+	mov r3, #1 ; MON_FEMALE
```

Comment:
It's normally not a good idea to document these asm files with comments since they will get deleted and replaced with their matching C files at some point.

Reply:
No problem, wasn't sure if that disassembly was supervised by a person or just done by machine so I figured I'd tend towards providing more information. Feel free to cut those commits out if needed :)

Reply:
I literally just decomped this function [here](https://github.com/pret/pokeheartgold/pull/100/files#diff-738c5949041a5b334c88329fc098a504b9edd4721d8ca82ad75e18aea614560cR172) lmao, though I did 100% miss the nature so will update that ty